bin/requirements.rb

```diff
@@ -90,8 +112,8 @@ def parse
         # skip git libraries. git>= line from vsphere gave us problems
         next if lib.start_with?("git")
 
-        # Do not version packages that are provided by the system: requests, requests[security], pyspnego[kerberos]
-        ver = "" if lib.match?(/^requests($|\[)/) || lib.start_with?("pyspnego")
+        # system packages are versioned by rpms
+        ver = "" if lib.match?(/^(#{OS_PACKAGES.join("|")})($|\[)/)
```

Comment:
Not really important, but not sure you knew about this one (although notice that it escaped the `-` in `cloud-what` correctly, which I didn't expect 😄)

```suggestion
        ver = "" if lib.match?(/^#{Regexp.union(OS_PACKAGES)}($|\[)/)
```

```ruby
[6] pry(main)> /^#{Regexp.union(OS_PACKAGES)}($|\[)/
=> /^(?-mix:six|dateutil|iniparse|idna|setuptools|inotify|libcomps|chardet|decorator|pysocks|urllib3|requests|cloud\-what|systemd|dbus|gobject|gpg|pyspnego)($|\[)/
```

Reply:
thanks - added to https://github.com/ManageIQ/manageiq-rpm_build/pull/509